dev-requirements.txt

```diff
@@ -8,4 +8,6 @@ vulture>=1.0
 bandit>=1.5.1
 pydocstyle>=3.0.0
 notebook>=5.2.2
+matplotlib>=3.3.0
```

Comment:
I think there is no problem with the lower versions, for example :
`matplotlib>=3.0.0`
`seaborn>=0.9.1`
Please double check it 💯

Reply:
Needed actions have been taken.